src/Map/MapDriver.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+namespace Map {
+    namespace Driver {
```

Comment:
just wondering why namespace over classes?

Reply:
Euhm.. no object creation necessary that way. Can be changed if ever.